mail_tracking/models/mail_message.py

```diff
@@ -35,8 +34,8 @@ def tracking_status(self):
         res = {}
         for message in self:
             partner_trackings = []
-            partners_already = self.env['res.partner']
-            partners = self.env['res.partner']
+            partners_already = []
```

Comment:
you don't need this list, the dict is enough. `lambda x: x.id not in partners` will work.